customs.js

```diff
@@ -55,6 +57,13 @@
       }
     )
     .then(
+      (function (result) {
+        if (result.lockout) {
+          return this.db.lockAccount(account)
+        }
+      }).bind(this)
+    )
+    .then(
       function () {},
       function (err) {
         log.error({ op: 'customs.flag.1', email: email, err: err })
```

Comment:
Good point about `customs.flag()` being called elsewhere too so I quite like the idea of having a re-usable chain of logic somewhere else ... as for where, wherever you think makes sense @shane-tomlinson. :)